test/config.json

```diff
@@ -12,7 +12,7 @@
     },
     {
       "file": "test/specs/multipleBrowsersDocumentUpload.js",
-      "timeout": 50000,
+      "timeout": 75000,
```

Comment:
Is this timeout adjustment based on the earlier JWT timeout value increase?

Reply:
No, this is because I've noticed that sometimes tests on browserstack are successful but they might take longer than a minute (in rare cases it might take even up to 2 mins). This means that our tests will fail while browserstack is still running the tests, So I decided to increase the timeout

Reply:
Wow! 2 mins is long. But we can always revisit this later if the current timeout is good enough.

Reply:
Two mins is rare but I've seen 1:13 quite often